test/e2e/framework/services/services.go

```diff
@@ -153,3 +143,28 @@ func StartServices(ctx context.Context, f *framework.Framework, mgr manager.Inte
 
 	return nil
 }
+
+func StartStorageSvc(ctx context.Context, f *framework.Framework, mgr manager.Interface) (storageSvcPort int, err error) {
+	storageDir, err := os.MkdirTemp("/tmp", "storagesvc")
+	if err != nil {
+		return 0, fmt.Errorf("error creating temp directory: %v", err)
```

Comment:
Please use %w instead. Same for all error message below

Reply:
updated

Reply:
done

Reply:
done